### ISSUE TITLE
Radio buttons do not work inside modal service displayed component

### ISSUE THREAD

**Issue by koolook** [valor-software/ngx-bootstrap#2294]
A simple component containing a couple of radio buttons
```html
    <div class="btn-group">
      <label class="btn btn-primary" [(ngModel)]="radioModel" btnRadio="Left">Left</label>
      <label class="btn btn-primary" [(ngModel)]="radioModel" btnRadio="Right">Right</label>
    </div>
```
is displayed using BsModalService.

Radio buttons does not seem to be working. When clicked
- view is not updated 
- model is not updated.

I compared their behavior to the same component displayed using directive and included into template.
1. Control initialization is correct
```JavScript
ButtonRadioDirective.writeValue(null)
ButtonRadioDirective.registerOnChange()
ButtonRadioDirective.registerOnTouched()
ButtonRadioDirective.writeValue(<initial value>)
```
for each button.

2. For clicked button
```
ButtonRadioDirective.onClick()
``` 
is triggered.

3. For service displayed component
```
ButtonRadioDirective.writeValue(<new value>)
```
is not triggered, when it should bring new value to each button.

Could not find the reason for 3 'cause debugging became too difficult.

**Comment by IlyaSurmay** [comment]
This will be fixed in the next release, 1.8.1

**Comment by koolook** [comment]
Great news, thanks! Looking forward to understand the reason for the bug.
